scripts/encrypt-secrets.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+echo "Enter password for encryption: "
+read password
+
+tar cvf secrets.tar testing/resources/{service-account.json,test-env.sh}
+openssl aes-256-cbc -k "$password" -in secrets.tar -out secrets.tar.enc
+rm secrets.tar
```

Comment:
So that we can decrypt, update the files, and re-encrypt as need. The password is stored internally. Now that our configuration is more than just a service account key, I feel it's pretty necessary.